test/beast/http/parser.cpp

```diff
@@ -357,6 +357,129 @@ class parser_test
         BEAST_EXPECT(p.need_eof());
     }
 
+    void
+    testMultipleContentLengthIssue1880()
+    {
+        {
+            // multiple contents lengths the same
+            const char header[] =
+                "HTTP/1.1 200 OK\r\n"
+                "Server: nginx\r\n"
+                "Date: Wed, 11 Mar 2020 17:18:19 GMT\r\n"
+                "Content-Type: application/octet-stream\r\n"
+                "Content-Length: 0\r\n"
+                "Connection: keep-alive\r\n"
+                "Access-Control-Allow-Origin: *\r\n"
+                "Access-Control-Allow-Methods: GET, OPTIONS, POST\r\n"
+                "Access-Control-Allow-Headers: Authorization, Content-Type, Accept\r\n"
+                "Access-Control-Max-Age: 1728000\r\n"
+                "Access-Control-Allow-Credentials: true\r\n"
+                "Content-Length: 0\r\n"
+                "Content-Type: text/plain\r\n"
+                "\r\n";
+
+            boost::beast::http::response_parser<boost::beast::http::string_body> parser;
```

Comment:
Why are all these names fully qualified??

Reply:
This code is copy/pasted from the MCVE we were given when the bug was reported.